bin/flutter-tizen

```diff
@@ -1,192 +1,76 @@
-#!/usr/bin/env python3
-# Copyright 2020 Samsung Electronics Co., Ltd. All rights reserved.
+#!/usr/bin/env bash
+# Copyright 2021 Samsung Electronics Co., Ltd. All rights reserved.
+# Copyright 2014 The Flutter Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
-import os
-import shutil
-import subprocess
-import sys
-import urllib.request
-import zipfile
-from platform import machine
-
-def revision(sourceDir):
-    command = ['git', 'rev-parse', 'HEAD']
-    return subprocess.run(
-        command, cwd=sourceDir, check=True, stdout=subprocess.PIPE
-    ).stdout.decode('utf-8').strip()
-
-def read(filePath):
-    if os.path.isfile(filePath):
-        with open(filePath, 'r') as f:
-            return f.read().strip()
-    return None
-
-def download(message, url, location):
-    size = 0
-    with urllib.request.urlopen(url) as res, open(location, 'wb') as f:
-        length = res.getheader('content-length')
-        total = f' of {int(length) / 1024 / 1024:.1f}' if length else ''
-        while True:
-            buf = res.read(64 * 1024)
-            if not buf:
-                break
-            size += f.write(buf)
-            print(f'{message} {size / 1024 / 1024:.1f}{total} MB\r', end='')
-        print(f'{message} Done' + ' ' * 15)
-
-def main():
-    if not machine().endswith('64'):
-        print('Not supported platform.')
-        exit(1)
-    if sys.platform.startswith(('cygwin', 'win')):
-        platform = 'windows'
-    elif sys.platform == 'darwin':
-        platform = 'darwin'
-    else:
-        platform = 'linux'
-
-    rootDir = os.path.abspath(__file__ + '/../..')
-    cacheDir = rootDir + '/bin/cache'
-    flutterDir = rootDir + '/flutter'
-    flutterRepo = 'https://github.com/flutter/flutter.git'
-
-    # Install flutter if not installed.
-    if os.path.isfile(flutterDir) or os.path.islink(flutterDir):
-        os.remove(flutterDir)
-    if not os.path.isdir(flutterDir):
-        cloneCommand = ['git', 'clone', '--depth=1', flutterRepo]
-        subprocess.run(cloneCommand, cwd=rootDir, check=True)
-
-    # Upgrade flutter if needed.
-    flutterVersionPath = rootDir + '/bin/flutter.version'
-    targetFlutterVersion = read(flutterVersionPath)
-
-    if targetFlutterVersion != revision(flutterDir):
-        subprocess.run(['git', 'reset', '--hard'], cwd=flutterDir, check=True)
-        subprocess.run(['git', 'clean', '-xdf'], cwd=flutterDir, check=True)
-        subprocess.run(['git', 'fetch', '--depth=1', flutterRepo,
-                        targetFlutterVersion], cwd=flutterDir, check=True)
-        subprocess.run(['git', 'checkout', 'FETCH_HEAD'],
-                       cwd=flutterDir, check=True)
-
-        # Invalidate the cache.
-        if os.path.isdir(cacheDir):
-            shutil.rmtree(cacheDir)
```

Comment:
You forgot to implement these lines in the new scripts.

Without these lines, flutter-tizen.snapshot will not be properly rebuilt even when the flutter SDK (flutter_tools) has been updated.

Also this could be the reason of the above error.